docs/mutable-stream.md

```diff
@@ -174,7 +174,7 @@ For One-Big-Table(OBT) or extra wide table with dozens or even hundreds of colum
 
 More commonly, you need to query a subset of the columns in different use cases. For those columns which are commonly queried together, you can define column families to group them, so that data for those columns will be saved together in the same file. Properly defining column families can optimize the disk i/o and avoid reading unnecessary data files.
 
-Please note, one column can appear up to one column family. The columns as primary keys are in a special column family. There should be no overlapping for the column families or primary keys.
+Please note, one column can appear in only one column family. The columns as primary keys are in a special column family. There should be no overlap for the column families or primary keys.
```

Comment:
`up to` is to highlight you don't have to put every column in one column family. Although `in only` roughly is wrong but I don't feel we have to change this